### PR TITLE
#104 - created reusable blogs card 

### PR DESCRIPTION
**Pull request by kire21**
- I used the existing blog card component for the blog page (made necessary changes to look like in the design).
- I made the tab component sticky, as well as the blog page layout. 

Screenshot from 2024-03-22 10-43-37
Screenshot from 2024-03-22 10-43-48

### CODE REVIEW

**Comment by stoimenov95** [comment]
Hey @kire21, the overall code looks fine to me. I just want to make sure regarding the design - should we put the two sections a bit closer to each other to remove that empty space we have now?
Screenshot 2024-03-22 at 11 20 38

**Comment by kire21** [comment]
> Hey @kire21, the overall code looks fine to me. I just want to make sure regarding the design - should we put the two sections a bit closer to each other to remove that empty space we have now? Screenshot 2024-03-22 at 11 20 38

Hi @stoimenov95,

We need to set the width for the body or main (ex. 1400px) in global.scss because now the layout is full width. When we set the width this problem will be fixed.

**Comment by edichoska** [comment]
> > Hey @kire21, the overall code looks fine to me. I just want to make sure regarding the design - should we put the two sections a bit closer to each other to remove that empty space we have now? Screenshot 2024-03-22 at 11 20 38
> 
> Hi @stoimenov95,
> 
> We need to set the width for the body or main (ex. 1400px) in global.scss because now the layout is full width. When we set the width this problem will be fixed.

Can we make the change only in the blogs layout, and adjust there? Meaning add margin-left?

**Comment by edichoska** [comment]
Screenshot 2024-03-22 at 11 39 33
Now depending on which tab we are, it shows beneath it За Нас or Најново. Please remove that.

**Comment by kire21** [comment]
> Screenshot 2024-03-22 at 11 39 33 Now depending on which tab we are, it shows beneath it За Нас or Најново. Please remove that.

Is it ok now?

image

**Review by edichoska — APPROVED** [review]
💅

**Review by stoimenov95 — CHANGES_REQUESTED** [review]

File: components/module-components/blog-list/BlogList.tsx [hunk 1]
```diff
@@ -19,14 +24,18 @@ const BlogList = async () => {
...
+            <BlogCard key={post?.id} title={post?.title} body={post?.body} pageTitle={pageTitle} />
+          );
         })}
       </div>
-      <Link href="/blog" className={style.blogBtn}>
-        Види повеќе <HiArrowLongRight fontSize={22} />
-      </Link>
+      {pageTitle === 'home' && (
+        <Link href="/blog" className={style.blogBtn}>
+          Види повеќе <HiArrowLongRight fontSize={22} />
```
[the remark this answers is not in the document]
Comment (edichoska) [reply components/module-components/blog-list/BlogList.tsx:36]: Hmm, lets do that, maybe https://icons.getbootstrap.com/ or they should research and find other acceptable options

[the remark this answers is not in the document]
Comment (edichoska) [reply components/module-components/blog-list/BlogList.tsx:36]: @stoimenov95 lets approve this one, and create tickets for the other things we discussed 🙏

**Comment by RevanchistX** [comment]
@kire21  Please include ticket number in the pull request name

**Review by stoimenov95 — CHANGES_REQUESTED** [review]
It looks a lot better now, great job @kire21!
I would only add some more spacing between the blogs section and the footer.

**Review by stoimenov95 — APPROVED** [review]

**Pull request merged**